build/verify-features.ts

```diff
@@ -1,45 +1,64 @@
 #!/usr/bin/env -S TS_NODE_COMPILER_OPTIONS='{"module":"es2020"}' node --loader ts-node/esm
 
-import {readdirSync} from 'node:fs';
+import {existsSync, readdirSync, readFileSync} from 'node:fs';
 
 import {getFeatures, getFeaturesMeta} from './readme-parser.js'; // Must import as `.js`
 
 const featuresDirContents = readdirSync('source/features/');
+const entryPoint = 'source/refined-github.ts';
+const entryPointSource = readFileSync(entryPoint);
 const importedFeatures = getFeatures();
 const featuresInReadme = getFeaturesMeta();
 
-const errors: string[] = [];
+function findError(filename: string): string | void {
+	if (filename === 'index.tsx') {
+		return;
+	}
+
+	if (filename.endsWith('.css')) {
```

Comment:
Can you extract this block of checks to its own function?

```suggestion
	if (filename.endsWith('.css')) {
      return findCssFileError(filename)
   }
```

Reply:
Addressed in 403512a2681737ae137a13c15fdf398a37dff591